tests/model/eventsources/test_api_event_source.py

```diff
@@ -118,31 +118,6 @@ def test_get_permission_with_just_slash(self):
 
         self.assertEqual(arn, "arn:aws:execute-api:${AWS::Region}:${AWS::AccountId}:${__ApiId__}/${__Stage__}/GET/")
 
-    @patch("boto3.session.Session.region_name", "eu-west-2")
-    def test_swagger_integration_with_string_rest_api_id(self):
-        self.api_event_source.Auth = {"Authorizer": "AWS_IAM", "ResourcePolicy": {"AwsAccountWhitelist": ["123456"]}}
-        self.api_event_source.to_cloudformation(
-            function=self.func,
-            explicit_api={
-                "__MANAGE_SWAGGER": True,
-                "DefinitionBody": {"openapi": "3.0.1", "paths": {"/foo": {}, "/bar": {}}},
-            },
-            intrinsics_resolver=IntrinsicsResolver({}),
-        )
-
-    @patch("boto3.session.Session.region_name", "eu-west-2")
-    def test_swagger_integration_with_ref_intrinsic_rest_api_id(self):
-        self.api_event_source.RestApiId = {"Ref": "ApiId"}
-        self.api_event_source.Auth = {"Authorizer": "AWS_IAM", "ResourcePolicy": {"AwsAccountWhitelist": ["123456"]}}
-        self.api_event_source.to_cloudformation(
-            function=self.func,
-            explicit_api={
-                "__MANAGE_SWAGGER": True,
-                "DefinitionBody": {"openapi": "3.0.1", "paths": {"/foo": {}, "/bar": {}}},
-            },
-            intrinsics_resolver=IntrinsicsResolver({}),
-        )
-
```

Comment:
The point of these was to test the api id, but it turned out that the api id wasn’t even used in `to_cloudformation`, after I went deeper into the functions where the api id was passed into. I removed this unused api id parameter (the other changes in this commit), and removed the unit tests because I don’t think it made sense to test the api id if it wasn’t used, and less so after removing it from the calls in `to_cloudformation`